swift/ql/src/queries/Summary/SummaryStats.ql

```diff
@@ -9,19 +9,42 @@
 import swift
 import codeql.swift.dataflow.FlowSources
 import codeql.swift.security.SensitiveExprs
+import codeql.swift.dataflow.DataFlow
+import codeql.swift.dataflow.TaintTracking
 
-predicate statistic(string what, int value) {
-  what = "Files" and value = count(File f)
+/**
+ * A taint configuration for tainted data reaching any node.
+ */
+class TaintReachConfig extends TaintTracking::Configuration {
+  TaintReachConfig() { this = "TaintReachConfig" }
+
+  override predicate isSource(DataFlow::Node node) { node instanceof FlowSource }
+
+  override predicate isSink(DataFlow::Node node) { any() }
+}
+
+float taintReach() {
+  exists(TaintReachConfig config, int tainted, int total |
+    tainted = count(DataFlow::Node n | config.hasFlow(_, n)) and
```

Comment:
Up until very recently `hasFlowTo(n)` was a simple alias for `hasFlow(_, n)`, but since https://github.com/github/codeql/pull/10744 `hasFlowTo` is actually a performance improvement since it doesn't compute the `fastTC` at all.
```suggestion
    tainted = count(DataFlow::Node n | config.hasFlowTo(n)) and
```

Reply:
Oh that's interesting!  I've never been that keen on `hasFlowTo` purely as a shortcut (`_` is perfectly clear), but if it has a performance benefit now I should get in the habit of using it consistently.